demo/src/examples/Scrollable.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import { merge } from 'lodash'
+import { Mention, MentionsInput } from '../../../src'
+
+import { provideExampleValue } from './higher-order'
+
+import defaultStyle from './defaultStyle'
+import defaultMentionStyle from './defaultMentionStyle'
+
+function Scrollable({ value, data, onChange, onAdd }) {
+  let style = merge({}, defaultStyle, {
+    highlighter: {
+      overflow: 'auto',
```

Comment:
This will cause two overlapping scrollbars to be shown. Set `overflow: 'hidden'` instead.

Reply:
We might set this in the general default styles of the `highlighter` element. Then it should be enough to just set the custom styles for the `input` element to make the componen scrollable.